arbnode/node.go

```diff
@@ -145,6 +146,7 @@ func (c *Config) ValidatorRequired() bool {
 
 func ConfigAddOptions(prefix string, f *flag.FlagSet, feedInputEnable bool, feedOutputEnable bool) {
 	f.Bool(prefix+".sequencer", ConfigDefault.Sequencer, "enable sequencer")
+	f.Uint64(prefix+".sync-till-block", ConfigDefault.SyncTillBlock, "sync till block")
```

Comment:
this setting should be explained better. node will not sync past this block.
It should be a config property of TransactionStreamer, which is the unit that creates the large sequence of blocks.
There is a good chance setting should be for message number and not block-number, though block-number is better for user experience and the offset between them is simple in the chain-config.

Reply:
I think block number is better.  If we are load balancing across multiple nodes, we typically know the block number

Reply:
Done.